next/site-config/anvil-portal/footer.ts

```diff
@@ -1,9 +1,70 @@
 import { FooterProps } from "@clevercanary/data-explorer-ui/lib/components/Layout/components/Footer/footer";
+import { SOCIAL } from "@clevercanary/data-explorer-ui/lib/components/common/Socials/socials";
+import logoHhs from "../../images/logoHhs.svg";
```

Comment:
We can now import images like this `import logoHhs from "images/logoHhs.svg";`

Reply:
Thanks for fixing this